plugins/modules/aci_maintenance_policy.py

```diff
@@ -218,16 +277,33 @@
 from ansible.module_utils.basic import AnsibleModule
 
 
+MATCH_RUN_MODE_MAPPING = dict(pause_always_between_sets="pauseAlwaysBetweenSets", pause_only_on_failures="pauseOnlyOnFailures", pause_never="pauseNever")
+MATCH_NOTIFY_CONDITION_MAPPING = dict(
+    notify_always_between_sets="notifyAlwaysBetweenSets", notify_never="notifyNever", notify_only_on_failures="notifyOnlyOnFailures"
+)
+MATCH_SMU_OPERATION_MAPPING = dict(smu_install="smuInstall", smu_uninstall="smuUninstall")
+MATCH_SMU_OPERATION_FLAGS_MAPPING = dict(smu_reload_immediate="smuReloadImmediate", smu_reload_skip="smuReloadSkip")
+
+
 def main():
     argument_spec = aci_argument_spec()
     argument_spec.update(aci_annotation_spec())
     argument_spec.update(
         name=dict(type="str", aliases=["maintenance_policy"]),  # Not required for querying all objects
-        runmode=dict(type="str", default="pauseOnlyOnFailures", choices=["pauseOnlyOnFailures", "pauseNever"]),
+        run_mode=dict(type="str", choices=list(MATCH_RUN_MODE_MAPPING.keys())),
         graceful=dict(type="bool"),
         scheduler=dict(type="str"),
-        ignoreCompat=dict(type="bool"),
-        adminst=dict(type="str", default="untriggered", choices=["triggered", "untriggered"]),
+        ignore_compat=dict(type="bool"),
```

Comment:
if you add alias in the other module, you might want to add here as well for consistency